src/content/plugins/internal-plugins.md

```diff
@@ -312,17 +285,3 @@ Merges chunks until each chunk has the minimum size of `minChunkSize`.
 `optimize/FlagIncludedChunksPlugin()`
 
 Adds chunk ids of chunks which are included in the chunk. This eliminates unnecessary chunk loads.
-
-### OccurenceOrderPlugin
-
-`optimize/OccurrenceOrderPlugin(preferEntry)`
-
-Order the modules and chunks by occurrence. This saves space, because often referenced modules and chunks get smaller ids.
-
-`preferEntry` If true, references in entry chunks have higher priority
-
-### DedupePlugin
```

Comment:
Removed in https://github.com/webpack/webpack/commit/67f31aeebc8b90d48e6733bea60d2203c1b8507d